jsonnet/openebs-mixin/README.md

```diff
@@ -0,0 +1,62 @@
+# openebs mixin
+
+A set of Grafana dashboards and Prometheus alerts for openebs.
+
+## Generate dashboards and rules
+
+You can manually generate the alerts, dashboards and rules files. 
+
+#### Steps:
+
+1. Install `jsonnet` and `jb`
+	```
+	$ go get github.com/jsonnet-bundler/jsonnet-bundler/cmd/jb
+	$ brew install jsonnet
+	```
+
+2. Then, grab the mixin and its dependencies:
+
+	```
+	$ git clone https://github.com/openebs/monitoring
+	$ cd monitoring/jsonnet/openebs-mixin
+	$ jb install
+	```
+
+3. Update the dashboards and rules directory path in `build.sh`
+	```
+	dashboardsDirPath = ''
+	rulesDirPath = ''
+	```
+
+4. Finally, build the mixin:
+
+	```
+	$ make generate
+	```
+
+The  files in `rulesDirPath` are the alert rules defined for openebs which has to be used to create prometheusRule yaml then need to passed to your Prometheus server, and the files in `dashboardsDirPath` need to be imported into you Grafana server. 
```

Comment:
need to be imported into your Prometheus.
need to be imported into your Grafana.

Reply:
The files (remove extra space between The and files) in `rulesDirPath` contain Prometheus-based alert rules for OpenEBS. These files will be aggregated to form a prometheusRule yaml that need to be imported into your Prometheus instance.
Similarly, the files in 'dashboardsDirPath' are dashboard json files that need to be imported into your Grafana instance.